.gitignore

```diff
@@ -373,3 +373,5 @@ target/
 
 # Cargo (Rust) lock file
 Cargo.lock
+.idea
+cmake-build-debug/
```

Comment:
Do we need these?

Reply:
Not necessarily, wouldn't hurt either, but can be removed.

Reply:
We can leave it. Thanks.